change/@microsoft-teams-js-252e0ab1-300c-4234-8b5c-76c706f3edec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Would we consider this a patch in that it's fixing a default runtime for mobile rather than adding a feature?

Reply:
I'm certainly open to suggestions here. For reference, this how beachball describes the difference:

Patch - bug fixes; no API changes.
Minor - small feature; backwards compatible API changes.

I will update it to "Patch"